### DISCUSSION TITLE
Is the typeahead field designed on purpose to delete selected entries when you hit backspace?

### DISCUSSION THREAD

**Discussion by ryanvelbon** [statamic/cms#6459] [Troubleshooting]
I have a typeahead field for my "Tags" taxonomy in my "Article" blueprint.

If I add some entries and then halfway through entering the next entry I change my mind and hit backspace to delete my input, once my input text is deleted any subsequeny backspace keypress will result in me accidentally deleting the tags I've already added.

Is there a way I can disable this functionality so that only my input text is deleted? It's pretty frustrating UX :D

**Comment by edalzell** [comment]
Sounds like a bug, can you please submit it?
